addons/web/static/src/js/widgets/date_picker.js

```diff
@@ -24,7 +24,8 @@ var DateWidget = Widget.extend({
         this._super.apply(this, arguments);
 
         this.name = parent.name;
-        this.options = _.extend({
+        var locale = (options.defaultDate && options.defaultDate._locale && options.defaultDate._locale._abbr) ? {locale: options.defaultDate._locale._abbr} : {}
```

Comment:
the [`property`](https://underscorejs.org/#property) function of underscore for multiple sub-accesses is available from v1.9, odoo uses 1.8

Reply:
Suggestion:
```
this.options = _.extend({
    locale: moment.locale(),
    // ...
}, ...);
```
It works even if there is no defaultDate, and we use moment API instead of doing introspection on a moment instance. What do you think?

Reply:
I think it is working and shorter ! :+1: